### ISSUE TITLE
Mesh Nodes statistic appears to be one short

### ISSUE THREAD

**Issue by m0xzl** [aredn/aredn#1545]
### Describe the bug
On the 'mesh' page I count 17 nodes. On the 'status' page it reports 16 nodes i.e. one less.

### Expected behavior
The 'nodes' statistic should match the number of nodes listed on the 'mesh' page.

### Screenshots
![image](https://github.com/user-attachments/assets/d187f110-fae5-40ac-9dc2-292bdab6ff1d)
![image](https://github.com/user-attachments/assets/3fa6b7b8-500f-4f09-a675-4741836b5e79)

### Additional context
* Have I misunderstood what the 'nodes' statistic is intended to represent?

**Comment by aanon4** [comment]
Could you attach the support data and confirm the above information at the time you download it (as these numbers can change)? The node count is generated by counting dtdlink entries in /var/run/hosts_olsr, and there should be one for each node. So I'm curious which node doesnt have one and why not.

**Comment by m0xzl** [comment]
[supportdata-M0XZL-GL5-2024-9-19-8-31.tar.gz](https://github.com/user-attachments/files/17055278/supportdata-M0XZL-GL5-2024-9-19-8-31.tar.gz)

**Comment by aanon4** [comment]
M0XZL-L1 appears to be different. Anything particular about it?

**Comment by m0xzl** [comment]
Yes, M0XZL-L1 is an old Linksys node running BBHN. It has been modified to provide a simple version of the SysInfo API. I leave it on the network because it is sometimes good to have a comparison for debugging code that I write to monitor things about the nodes.

I believe it is M7SHF-GL1 that is missing from the hosts_olsr file.

**Comment by aanon4** [comment]
M7SHF-GL1 is there.
<img width="438" alt="Screenshot 2024-09-19 at 12 55 58 AM" src="https://github.com/user-attachments/assets/73657427-efae-4243-8009-1504d3c9b786">

**Comment by aanon4** [comment]
So I guess the node count is correct, because one of the devices is not an AREDN node (in the way we count them)

**Comment by m0xzl** [comment]
OK, I can cope with that now that I understand the difference.
